### PR TITLE
docs: modifying doc on vmMemoryOverheadPercent

### PR DESCRIPTION
**Pull request by fedecz**
Fixes [#6612 ](https://github.com/aws/karpenter-provider-aws/issues/6612)

**Description**

Just a minor change in the docs with a simple explanation. Happy to revisit the wording if required. 

**How was this change tested?**

By running `make website` and checking the changed line is correct.

**Does this change impact docs?**
- [x] Yes, PR includes docs updates 
- [ ] Yes, issue opened: # 
- [ ] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/preview/reference/settings.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Karpenter surfaces environment variables and CLI parameters to allow you to conf
 | MEMORY_LIMIT | \-\-memory-limit | Memory limit on the container running the controller. The GC soft memory limit is set to 90% of this value. (default = -1)|
 | METRICS_PORT | \-\-metrics-port | The port the metric endpoint binds to for operating metrics about the controller itself (default = 8080)|
 | RESERVED_ENIS | \-\-reserved-enis | Reserved ENIs are not included in the calculations for max-pods or kube-reserved. This is most often used in the VPC CNI custom networking setup https://docs.aws.amazon.com/eks/latest/userguide/cni-custom-network.html. (default = 0)|
-| VM_MEMORY_OVERHEAD_PERCENT | \-\-vm-memory-overhead-percent | The VM memory overhead as a percent that will be subtracted from the total memory for all instance types. (default = 0.075)|
+| VM_MEMORY_OVERHEAD_PERCENT | \-\-vm-memory-overhead-percent | The VM memory overhead as a percent that will be subtracted from the total memory for all instance types. (default = 0.075, which equals to 7.5%)|
```
Comment (engedaam) [inline website/content/en/preview/reference/settings.md:35]: Can you make this change to all the version of the docs?

Comment (fedecz) [reply website/content/en/preview/reference/settings.md:35]: sure, hang on.

Comment (fedecz) [reply website/content/en/preview/reference/settings.md:35]: done @engedaam

**Review by engedaam — APPROVED** [review]
LGMT 🚀

File: website/content/en/preview/reference/settings.md (hunk 1 above)
Comment (engedaam) [reply website/content/en/preview/reference/settings.md:35]: Thank you!

**Pull request merged**